components/HubHeader.vue

```diff
@@ -154,7 +154,7 @@ export default class extends Vue {
   }
 
   reset() {
-      console.log("here");
+    console.log("here");
```

Comment:
Can we remove console logs?

Reply:
There is also one in "updated" function, shall I remove that one too?

Reply:
Done!